doc/api/tls.markdown

```diff
@@ -764,6 +764,8 @@ of written data and all required TLS negotiation.
 This instance implements a duplex [Stream][] interfaces.  It has all the
 common stream methods and events.
 
+Methods that return TLS connection meta data (e.g. `getPeerCertificate()`) will only return data while the connection is open.
```

Comment:
Updated to use correct column length.

I couldn't get intra-page links working for that section. I tried using the markdown `[getPeerCertificate](#tlssocketgetpeercertificate-detailed-)` and when I opened the preview in github and clicked on it, it didn't work. However typing in the url directly (`https://github.com/DaftMonk/node/blob/docs-tls-socket/doc/api/tls.markdown#tlssocketgetpeercertificate-detailed-`) did work. So I'm not sure what the issue was, but I reverted it back to a code example.